### PR TITLE
[WIP]modulesync 0.6.1

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
does anybody want to review?

currently only spec tests for concat::fragments fail. They look very strange and I'm not sure about the best way to check them. 

CC @igalic @jyaworski

**Comment by jyaworski** [comment]
It looks like we're using a newer rspec. Is subject explicitly defined anywhere?

File: spec/classes/dhcp_spec.rb [hunk 1]
```diff
@@ -253,7 +253,7 @@
         )
       end
       it { is_expected.to contain_concat__fragment('dhcp-conf-header').with_content %r{^option root-path "/opt/ltsp/i386";$} }
-      it { is_expected.to contain_concat__fragment('dhcp-conf-header').with_content %r{^option tftp-server-name "1.2.3.4";$} }
+      it { is_expected.to contain_concat__fragment('dhcp-conf-header').with_content(/^option tftp-server-name "1\.2\.3\.4";$/) }
```
Comment (alexjfisher) [inline spec/classes/dhcp_spec.rb]: I'm not sure I really like changing `%r{}` to `//` here...

**Comment by bastelfreak** [comment]
the 4 rubocop fails will be handled by the next modulesync

File: manifests/host.pp [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
   $host = $name
 
-  include dhcp::params
+  include ::dhcp::params
```
Comment (alexjfisher) [inline manifests/host.pp]: @bastelfreak Any reason behind not inheriting the params class here like you did for failover.pp and disable.pp?

Comment (bastelfreak) [inline manifests/host.pp]: params.pp pattern works only for classes, not for defined resources.

Comment (alexjfisher) [inline manifests/host.pp]: Ah yes, good point!
How about `$dhcp_dir = $::dhcp::dhcp_dir` instead?

Comment (bastelfreak) [inline manifests/host.pp]: could you test that? I thought that doesn't work on puppet3 with STRICT_VARIABLES enabled?

Comment (alexjfisher) [inline manifests/host.pp]: Something like https://github.com/voxpupuli/puppet-dhcp/pull/82 ?

**Pull request merged**